src/components/NavBar/utils.ts

```diff
@@ -30,25 +36,38 @@ type LayersCategoriesType = LayersCategoryType[];
 type MenuItemsType = MenuItemType[];
 
 function formatLayersCategories(layersList: {
-  [key: string]: string[];
+  [key: string]: Array<LayerKey | TableKey>;
 }): LayersCategoriesType {
-  return map(layersList, (layerKeys, layersListKey) => ({
-    title: startCase(layersListKey),
-    layers: layerKeys
-      .map(key => LayerDefinitions[key])
-      .filter((val): val is LayerType => Boolean(val)),
-
-    tables: layerKeys
-      .map(key => TableDefinitions[key])
-      .filter(val => Boolean(val)),
-  }));
+  return map(layersList, (layerKeys, layersListKey) => {
```

Comment:
Any reason not to just use the builtin `map` here?

Reply:
are you referring to the `Array#map` here? `layersList` is an object, hence the use of `map` as a shorthand to `Object.entries().map()` which is the only vanilla methodology I can think of for this case. If you know a better vanilla way let me know :)

Reply:
Ya, it'd require `Object.entries` (or `Object.keys/Object.values`) and then `map`. But generally I don't think that's a big deal to add that one call, and the (theoretical at this point) idea is to only use Lodash where it's strictly necessary/a big savings over what is provided in the standard library. In theory this means that we could use tree-shaking to selectively include only parts of Lodash in the build, rather than the whole library.